extensions/landing-zone/kubeconfig/README.md

```diff
@@ -0,0 +1,2 @@
+This directory must exist in source control so the `ibm_container_cluster_config` data lookup can use it to place the
```

Comment:
Is this comment going to persist? And who sees it? We probably don't want to point to an internal issue from this module.